imagehash.py

```diff
@@ -89,7 +93,7 @@ def __init__(self, binary_array):
 		self.hash = binary_array
 
 	def __str__(self):
-		return _binary_array_to_hex(self.hash.flatten())
+		return _binary_array_to_hex(self.hash)
```

Comment:
why flatten here instead of in the other function?

Reply:
> Would this work OK with very large hashes (e.g., 64x64)?

Yes, it seems to keep the advantage on large hashes, too. Converting 3000 randomly generated 64x64 hashes gives the following results:
```
Original version: 15.335927000000083
Updated version: 7.23339599999963
```

It only gives in at about 160-dimensional hashes (25000+ bits). Running 1000 160x160 hashes gives a rough parity:
```
Original version: 37.50718020000204
Updated version: 37.62010579999696
```

Reply:
I found that `nditer()` function iterates over all elements of multidimensional arrays already, so calling `flatten()` is not necessary here, or perhaps even harmful due to the cost of array copying.